include/godot_cpp/classes/wrapped.hpp

```diff
@@ -95,9 +95,15 @@ class Wrapped {
 	GodotObject *_owner = nullptr;
 };
 
+GDExtensionPropertyInfo *create_c_property_list(const ::godot::List<::godot::PropertyInfo> &plist_cpp, uint32_t *r_size);
+void free_c_property_list(GDExtensionPropertyInfo *plist);
```

Comment:
These functions are in the `godot` namespace and lots of folks do `using namespace godot`, whereas these aren't really API functions that we want people to call.

Could these be put in the `godot::internal` namespace instead?

Reply:
Added it

Reply:
Thanks!